Makefile

```diff
@@ -44,7 +45,7 @@ build:
 depsdev:
 	go install github.com/Songmu/ghch/cmd/ghch@latest
 	go install github.com/Songmu/gocredits/cmd/gocredits@latest
-	go install github.com/securego/gosec/v2/cmd/gosec@latest
+	go install golang.org/x/vuln/cmd/govulncheck@latest
```

Comment:
Since you no longer use gosec, do you no longer need to suppress the following warning?
https://vscode.dev/github/k1LoW/runn/blob/enable-configurable-http-runner-timeout/operator.go#L1356

Reply:
gosec is actually running in golanci-lint 👍 

https://github.com/k1LoW/runn/blob/e7456e1e6ad594e630476ca80abdaf46844cbd86/.golangci.yml#L5

Reply:
nrhd-